modules/gitops-argocd-cli-synchronizing-an-application-in-normal-mode.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * argocd_applications/synchronizing-an-application-using-gitops-argocd-cli.adoc
+// * declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+ifeval::["{context}" == "configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations"]
+:cluster:
+endif::[]
+ifeval::["{context}" == "synchronizing-an-application-using-gitops-argocd-cli"]
+:app:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argocd-cli-synchronizing-an-application-in-normal-mode_{context}"]
+= Synchronizing an application in normal mode by using the GitOps CLI
+
+You can synchronize applications in normal mode by using the {gitops-shortname} `argocd` CLI.
+
+ifdef::cluster[]
+This sample workflow walks you through the process of configuring Argo CD to recursively sync the content of the `cluster` directory to the `cluster-configs` application. The directory defines the {OCP} cluster configurations and a namespace `spring-petclinic` on the cluster.
+endif::cluster[]
+
+.Prerequisites
+
+* You have installed the {gitops-title} Operator in your cluster.
+* You have logged into Argo CD instance.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {gitops-title} `argocd` CLI. 
+
+.Procedure
+
+. Get the admin account password for the Argo CD server:
++
+[source,terminal]
+----
+$ ADMIN_PASSWD=$(oc get secret openshift-gitops-cluster -n openshift-gitops -o jsonpath='{.data.admin\.password}' | base64 -d)
+----
+
+. Get the Argo CD server URL:
++
+[source,terminal]
+----
+$ SERVER_URL=$(oc get routes openshift-gitops-server -n openshift-gitops -o jsonpath='{.status.ingress[0].host}')
+----
+
+. Log in to the Argo CD server:
++
+.. Display the line of text or string of the admin account password:
++
+[source,terminal]
+----
+$ echo ${ADMIN_PASSWD}
```

Comment:
Same comment as above

Reply:
@varshab1210 I guess it is to rule out the characters like $ in the password. There is also a note provided explaining the case and situation.

@anandf Can you please comment on this and provide the clarification?

Reply:
Removing the `echo` step. Customers can use the placeholder field and paste the password enclosed in single quotes.